build/test/integration/distribution_test.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-/* global require, process, suite, setup, test, teardown */
-/* jshint -W101 */
+/* jshint maxlen: 130 */
```

Comment:
Some code snippets over 80 characters in https://bugzilla.mozilla.org/show_bug.cgi?id=1132225#c15
I prefer to extend max length of line checking rather than wrap it to a new line.